README.md

```diff
@@ -32,7 +33,7 @@ MRTK-Unity is a Microsoft-driven project that provides a set of components and f
 
 # Feature areas
 
-| ![Input System](Documentation/Images/MRTK_Icon_InputSystem.png) [Input System](Documentation/Input/Overview.md)<br/>&nbsp;  | ![Hand Tracking<br/> (HoloLens 2)](Documentation/Images/MRTK_Icon_HandTracking.png) [Hand Tracking<br/> (HoloLens 2)](Documentation/Input/HandTracking.md) | ![Eye Tracking<br/> (HoloLens 2)](Documentation/Images/MRTK_Icon_EyeTracking.png) [Eye Tracking<br/> (HoloLens 2)](Documentation/EyeTracking/EyeTracking_Main.md) | ![Profiles](Documentation/Images/MRTK_Icon_Profiles.png) [Profiles](Documentation/MixedRealityConfigurationGuide.md)<br/>&nbsp; | ![Gaze + Gesture<br/> (HoloLens)](Documentation/Images/MRTK_Icon_GazeSelect.png) [Gaze + Gesture<br/> (HoloLens)](Documentation/Input/Gestures.md) |
+| ![Input System](Documentation/Images/MRTK_Icon_InputSystem.png) [Input System](Documentation/Input/Overview.md)<br/>&nbsp;  | ![Hand Tracking<br/> (HoloLens 2)](Documentation/Images/MRTK_Icon_HandTracking.png) [Hand Tracking<br/> (HoloLens 2)](Documentation/Input/HandTracking.md) | ![Eye Tracking<br/> (HoloLens 2)](Documentation/Images/MRTK_Icon_EyeTracking.png) [Eye Tracking<br/> (HoloLens 2)](Documentation/EyeTracking/EyeTracking_Main.md) | ![Profiles](Documentation/Images/MRTK_Icon_Profiles.png) [Profiles](Documentation/MixedRealityConfigurationGuide.md)<br/>&nbsp; | ![Hand Tracking<br/> (Ultraleap)](Documentation/Images/MRTK_Icon_HandTracking.png) [Hand Tracking (Ultraleap)](Documentation/CrossPlatform/LeapMotionMRTK.md)|
```

Comment:
It feels like it would be better to have a single hand tracking cell, where we put down both HL2 + Leap stuff both covered/linked (i.e. one for hand tracking specifically which then goes to how to get started with HL2 or Leap).

I think this would require some refactoring of content, however, so this is probably fine for the time being.